### PR TITLE
Adding Find Subarrays With Equal Sum

### PR DESCRIPTION
**Pull request by hot9cups**
- Adding the Leetcode problem 'Find Subarrays With Equal Sum' with corresponding Test File.
- Minor readme formatting fix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,8 @@ To run a specific problem in your console run  `node <problem_file_path>` (e.g.
...
 | [Lowest Common Ancestor of a Binary Tree ](/LeetcodeProblems/Algorithms/Lowest_Common_Ancestor_of_a_Binary_Tree.js) | Medium | https://leetcode.com/problems/lowest-common-ancestor-of-a-binary-tree/ |
 | [Maximum Sum of an Hourglass](/LeetcodeProblems/Algorithms/Maximise_Hour_Glass_Sum.js)                          | Medium | https://leetcode.com/problems/maximum-sum-of-an-hourglass/ |
-| [Time Needed to Rearrange a Binary String] (/LeetcodeProblems/Algorithms/Time_Needed_Rearrange_Binary_String.js)| Medium | https://leetcode.com/problems/time-needed-to-rearrange-a-binary-string/ |
+| [Time Needed to Rearrange a Binary String](/LeetcodeProblems/Algorithms/Time_Needed_Rearrange_Binary_String.js)| Medium | https://leetcode.com/problems/time-needed-to-rearrange-a-binary-string/ |
+| [Find Subarrays With Equal Sum ](/LeetcodeProblems/Algorithms/Find_Subarrays_With_Equal_Sums.js)                                                                   | Medium | https://leetcode.com/problems/find-subarrays-with-equal-sum/ |
```
Comment (ignacio-chiazzo) [inline README.md:66]: Could you ident this?

**Review by ignacio-chiazzo — APPROVED** [review]
Hey! it looks good to me. Thanks!

File: README.md (hunk 1 above)
Comment (hot9cups) [reply README.md:66]: Hm, I tried to follow the same format as the others in the readme. Indent what, could you explain please? The preview of the readme looks fine so I'm maybe missing something

Comment (ignacio-chiazzo) [reply README.md:66]: It might be github but I see the | medium | not aligned. Anyways, this is good to merge, thank you!

**Pull request merged**